src/main/java/calculator/add/string/model/Numbers.java

```diff
@@ -0,0 +1,20 @@
+package calculator.add.string.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Numbers {
+
+    private final List<Number> numbers;
+
+    public Numbers(String[] numbers) {
+        this.numbers = Arrays.stream(numbers).map(Number::new).collect(Collectors.toCollection(ArrayList::new));
+    }
```

Comment:
```suggestion
    public Numbers(String[] numbers) {
        this.numbers = Arrays.stream(numbers)
                .map(Number::new)
                .collect(Collectors.toList());
    }
```
메소드 체이닝할때 개행을 해주면 조금 더 가독성이 좋아지는거같더라구요